src/components/widgets/NumberWidget.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import {IPoint} from 'types/IPoint';
+import {LoadingOverlay} from 'components/LoadingOverlay/LoadingOverlay';
+import styles from "./shared/BasicNumberWidgetStyle.module.css";
```

Comment:
In this case, since you've added a css file, you should put `NumberWidget.tsx` into a folder titled `NumberWidget` inside of `src/components/widgets`, and put `BasicNumberWidgetStyle.module.css` next to `NumberWidget.tsx`

Also, to be consistent with naming, you should change `BasicNumberWidgetStyle.module.css` -> `NumberWidget.module.css`

See attached photo for example.

![image](https://user-images.githubusercontent.com/29921020/59153876-6e948f00-8a2a-11e9-99f9-2522022561ba.png)

Reply:
Done!